src/inverseRobot.js

```diff
@@ -5,9 +5,20 @@
  *
  * @return {object}
 */
-
 function inverseRobot(robot) {
-  // write code here
+  let inversedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
```

Comment:
Try to come up with more descriptive variable name here, and everything else is great!